src/components/Dashboard/Alerts.vue

```diff
@@ -51,6 +54,7 @@ export default class Alerts extends Vue {
   /** The list of alerts. */
   get alerts (): Array<any> {
     const alerts = []
+    if (this.isInDissolutionAlert) alerts.push(InDissolution) // needs to be shown first (high priority)
```

Comment:
From the designs, this alert will precede all others if it exists. We want to show it first.